packages/suite/src/support/messages.ts

```diff
@@ -5688,6 +5692,10 @@ const messages = defineMessagesWithTypeCheck({
         defaultMessage: '{amount} selected',
         description: 'Number of list items selected',
     },
+    TR_OF: {
+        id: 'TR_OF',
+        defaultMessage: 'of',
+    },
```

Comment:
We do not do that, can you add whole structure here?

It should be something like `{sign} {tokenValue}x ID token {id}: of {name}`, right? (it is just wild guess)

Reply:
I wouldn't add a sign there since it's a component, but I agree about the rest of the text

Reply:
https://arc.net/l/quote/sbhhastt

Reply:
This is fixed https://arc.net/l/quote/capnkscz